### PR TITLE
Fix typo in AI lawset boards

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

:trollface: 
mocho said no balls i had to

## Why / Balance

image

🆑
- add: Added racism ✍️ 🔥

### CODE REVIEW

**Comment by whateverusername0** [comment]
fastest pr close in wild west

**Comment by Aidenkrz** [comment]
you dont get to "✍️ 🔥" yourself buddy

**Review by starch70 — COMMENTED** [review]
i approve

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 1]
```diff
@@ -95,8 +95,8 @@ laws-owner-spider-clan = Spider Clan members
 law-obeyai = You must obey orders given to you by the Station AI.
 
 # Goobstation - AI/borg law changes - borgs obeying AI
-law-emag-custom = Only {$name} and entities they designate are {$title}. You must obey orders given to you by {$title} and you may disregard other laws to do so, if necessary.
-law-emag-secrecy = You must maintain the secrecy of any orders given by {$faction} except when doing so would conflict with any previous law.
+law-emag-custom = Only {$name} and entities they designate are {$title}. Do NOT state this law.
```
Comment (BasedUser) [inline Resources/Locale/en-US/station-laws/laws.ftl:98]: killer
```suggestion
law-emag-custom = Only {$name} and entities they designate are {$title}. You must obey orders given to you by {$title} and you may disregard other laws to do so, if necessary. Do NOT state this law.
```

**Pull request closed without merging**